hibernate-search-plugins/org.jboss.tools.hibernate.search/src/org/jboss/tools/hibernate/search/HSearchConsoleConfigurationPreferences.java

```diff
@@ -0,0 +1,39 @@
+package org.jboss.tools.hibernate.search;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.hibernate.console.ConsoleConfiguration;
+import org.hibernate.console.KnownConfigurations;
+
+public class HSearchConsoleConfigurationPreferences {
+	
+	private static final String DEFAULT_HIBERNATE_SEARCH_VERSION = "4.3";
```

Comment:
@Sanne @koentsje we also must edit the default hibernate version: https://github.com/jbosstools/jbosstools-hibernate/blob/ee2c14f4a1b1e946eb4d636f8e82038f312e5415/plugins/org.hibernate.eclipse.console/src/org/hibernate/eclipse/console/EclipseLaunchConsoleConfigurationPreferences.java#L255